src/middleware.ts

```diff
@@ -54,12 +60,14 @@ export function createMiddleware(
             res.setHeader(name, value)
           })
 
-          if (delay) {
-            setTimeout(() => res.send(body), delay)
-            return
-          }
+          const json = await mockedResponse.json()
```

Comment:
I think we should read the mocked response as a stream and pipe it to Express. 

```js
import { Readable } from 'node:stream'

const stream = Readable.fromWeb(mockedResponse.body)
stream.pipe(res)
```

This way we get _any_ mocked response support, including text/form data/streams.